src/status_im/accounts/recover/core.cljs

```diff
@@ -110,21 +118,20 @@
             (assoc :accounts/new-installation-id (random-guid-generator)))}
    (node/initialize nil)))
 
-(fx/defn recover-account-with-checks [{:keys [db] :as cofx}]
-  (let [{:keys [passphrase processing?]} (:accounts/recover db)]
-    (when-not processing?
-      (if (mnemonic/status-generated-phrase? passphrase)
-        (recover-account cofx)
-        {:ui/show-confirmation
-         {:title               (i18n/label :recovery-typo-dialog-title)
-          :content             (i18n/label :recovery-typo-dialog-description)
-          :confirm-button-text (i18n/label :recovery-confirm-phrase)
-          :on-accept           #(re-frame/dispatch [:accounts.recover.ui/recover-account-confirmed])}}))))
-
-(fx/defn navigate-to-recover-account-screen [{:keys [db] :as cofx}]
+(fx/defn navigate-to-recover-account-screen [{:keys [db] :as cofx} phase]
   (fx/merge cofx
             {:db (dissoc db :accounts/recover)}
-            (navigation/navigate-to-cofx :recover nil)))
+            (navigation/navigate-to-cofx :recover (assoc cofx :phase phase))))
+
+(fx/defn navigate-to-recover-account-password-screen [{:keys [db] :as cofx}]
+  (fx/merge cofx
+            {:db (assoc db :intro-wizard {:step :create-code
+                                          :weak-password? true
+                                          :encrypt-with-password? true
+                                          :accounts (cofx :accounts)
+                                          :recovery? true
+                                          :masked-passphrase (cofx :masked-passphrase)})}
+            (navigation/navigate-to-cofx :intro-wizard cofx)))
```

Comment:
Extra `cofx` parameter is not required, as `cofx` is already passed as a first param to `navigate-to-cofx` by `fx/merge` macro.

Reply:
Done!